### ISSUE TITLE
Text formatting in Corfunc Perspective needs attention

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#2456]
Copied from PR https://github.com/SasView/sasview/pull/2450

As far as I can see, the parameter part of the Corfunc perspective is of fixed width; you can drag the who perspective wider, but all that does is stretch the plots, you can't widen the parameter part? The reason I was trying to do this is that on my monitor (standard 1920x1080 widescreen) the text for some of the Lamellar Parameter boxes is 'lost':
![image](https://user-images.githubusercontent.com/10938679/223777887-6f6980e4-b03a-4e7c-be41-8c46e60cb2e5.png)

**Comment by lucas-wilkins** [comment]
Maybe it does, but it is probably the general issue with font scaling.

**Comment by smk78** [comment]
So I thought I would test if it was the general font scaling whilst I was evaluating https://github.com/SasView/sasview/pull/2463.
What I did was go to the new File > Preferences > Display Options and changed the scaling to either 'automatic' or unchecked 'automatic' and set a scale of 0.9. I then clicked 'Apply' at which point I was informed the program needed to restart. So I ok'd that and this happened  in both instances:
![image](https://user-images.githubusercontent.com/10938679/225698688-36a547c1-0abf-4325-a2f4-21afcc181395.png)
